DeepFilterNet/df/scripts/export.py

```diff
@@ -303,12 +304,17 @@ def main(args):
     )
     sample = get_test_sample(df_state.sr())
     enhanced = enhance(model, df_state, sample, True)
-    save_audio("out/enhanced.wav", enhanced, df_state.sr())
-    if not os.path.isdir(args.export_dir):
-        os.makedirs(args.export_dir)
+    try:
+        # attempt saving enhanced audio
+        save_audio("out/enhanced.wav", enhanced, df_state.sr())
+    except RuntimeError:
+        pass
+    export_dir = Path(args.export_dir)
+    if not export_dir.is_dir():
```

Comment:
This is not needed due to `exist_ok=True`

Reply:
Yep, I'll remove the `if` statement

Reply:
Changed in bd3659b9bd4af1678340224cb7833d33bdd4a0bc